### ISSUE TITLE
Serial causes access violation OpenJDK11

### ISSUE THREAD

**Issue by tresf** [qzind/tray#414]
Using OpenJDK11 from AdoptOpenJDK on Windows with jssc causes an access violation,

Possible upstream duplicate(s):
https://github.com/scream3r/java-simple-serial-connector/issues/116
https://github.com/scream3r/java-simple-serial-connector/issues/125
https://github.com/scream3r/java-simple-serial-connector/issues/124

```
        Parsed serial setting: FLOWCONTROL_NONE
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x000000007110b5db, pid=5884, tid=5976
#
# JRE version: OpenJDK Runtime Environment (11.0.2+9) (build 11.0.2+9)
# Java VM: OpenJDK 64-Bit Server VM (11.0.2+9, mixed mode, tiered, compressed oops, g1 gc, windows-amd64)
# Problematic frame:
# C  [jSSC-2.8_x86_64.dll+0xb5db]
#
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# C:\Users\owner\hs_err_pid5884.log
#
# If you would like to submit a bug report, please visit:
#   https://github.com/AdoptOpenJDK/openjdk-build/issues
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```

hs_err_pid5884.log attached: [hs_err_pid5884.log](https://github.com/qzind/tray/files/2848329/hs_err_pid5884.log)

**Comment by tresf** [comment]
Closed via a85e045b0d7ef39a99d84ff550a9fa4f3df57a41.

**Comment by Toro98M** [comment]
What was the solution?

**Comment by tresf** [comment]
@Toro98M the fix was upgrading to JSSC 1.9.1 per https://github.com/qzind/tray/commit/a85e045b0d7ef39a99d84ff550a9fa4f3df57a41.

Note, due to lack of response from upstream, this 1.9.1 is our own forked copy of JSSC.
